src/constants.ts

```diff
@@ -374,6 +374,20 @@ export enum TelemetryEventName {
     MICROBIT_PERFORMANCE_DEPLOY_DEVICE = "MICROBIT.PERFORMANCE.DEPLOY.DEVICE",
     MICROBIT_PERFORMANCE_NEW_FILE = "MICROBIT.PERFORMANCE.NEW.FILE",
     MICROBIT_PERFORMANCE_OPEN_SIMULATOR = "MICROBIT.PERFORMANCE.OPEN.SIMULATOR",
+
+    // Venv options
+    SETUP_VENV_CREATION_ERR = "SETUP.VENV.CREATION.ERR",
+    SETUP_NO_PIP = "SETUP.NO.PIP",
+    SETUP_DEP_INSTALL_FAIL = "SETUP.DEP.INSTALL.FAIL",
+    SETUP_AUTO_RESOLVE_PYTHON_PATH = "SETUP.AUTO.RESOLVE.PYTHON.PATH",
+    SETUP_NO_PYTHON_PATH = "SETUP.NO.PYTHON.PATH",
+    SETUP_DOWNLOAD_PYTHON = "SETUP.DOWNLOAD.PYTHON",
+    SETUP_BAD_PYTHON_PATH = "SETUP.BAD.PYTHON.PATH",
```

Comment:
nit:can we change it for wrong python path instead of bad python path

Reply:
Bad python path pertains to having an interpreter path to a file that does not exist. I will try to change the name to make it make more sense!